### PR TITLE
Add ui to bundle

### PR DESCRIPTION
**Pull request by francisco-dlp**
Distribute HyperSpyUI with the Windows Bundle

### CODE REVIEW

**Comment by vidartf** [comment]
Hyperspyui has a dependency on hyperspy, so I think the UI install needs to come after the hyperspy install to get things in the right order. I'll try to test this further tomorrow.

**Comment by francisco-dlp** [comment]
Done, thanks!

**Comment by vidartf** [comment]
Will test it in action tomorrow.

File: appveyor.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@ before_deploy:
   - "%CMD_IN_ENV% python.exe -c \"import sys; print(sys.version)\""
   - "%CMD_IN_ENV% python.exe setup.py recythonize"
   - "%CMD_IN_ENV% python.exe setup.py install"
+  - "%CMD_IN_ENV% pip install --upgrade hyperspyui"
```
Comment (vidartf) [inline appveyor.yml]: It seems the `--upgrade` flag is causing some problems: https://ci.appveyor.com/project/vidartf/hyperspy/build/1.2.0.358/job/atqrbqwvka3b1k4c

**Comment by francisco-dlp** [comment]
Fixed!

**Comment by vidartf** [comment]
One more change needed, due to some strange errors with pip:

```diff
-  - "%CMD_IN_ENV% pip install hyperspyui"
+  # Try to run twice as workaround for permission error
+  - "%CMD_IN_ENV% pip install hyperspyui || pip install hyperspyui"
```

**Comment by francisco-dlp** [comment]
Feel free to push this and any other required changes directly to this branch (no PR required).

**Comment by vidartf** [comment]
I was originally a bit concerned about tag contamination, but with the new direct pushing, it should be safe (since tags only go up when you use `git push --tags`).

**Comment by vidartf** [comment]
Changes pushed. For an appveyor run with these changes (and a tag), see https://ci.appveyor.com/project/vidartf/hyperspy/build/1.2.0.380 . The resulting bundles are available from the artifacts tab under the different jobs.

**Comment by francisco-dlp** [comment]
I've just tried it. The desktop integration didn't work for me but running from terminal works and it's great!

**Comment by vidartf** [comment]
Is the UI integration something that needs to be fixed in this package or the bundle package?

**Comment by vidartf** [comment]
PS: Any idea why the `test_binary_string` test is failing on Travis? 😕

**Comment by francisco-dlp** [comment]
Yes, it's due to dill failing to read functions across Python versions. I'm working on a fix.

You're right about the desktop integration, it should be fixed elsewhere.

**Pull request merged**